pkg/controller/daemon/daemon_controller.go

```diff
@@ -1290,6 +1333,14 @@ func (o podByCreationTimestampAndPhase) Less(i, j int) bool {
 	return o[i].CreationTimestamp.Before(&o[j].CreationTimestamp)
 }
 
+func podNames(pods []*v1.Pod) []string {
```

Comment:
This doesn't appear to be used anywhere yet, consider deleting?

Reply:
Yup, will be nuked